### ISSUE TITLE
Try To Assign Mod ID For Manual Installs

### ISSUE THREAD

**Issue by effs** [E1337Kat/cyberpunk2077_ext_redux#95]

**Comment by effs** [comment]
> Mind providing some more context? Like expected vs actual?

Yes, thanks, I just left this for  myself as a memo before I'd forget :)

**Comment by Bladehawke** [comment]
Yes, the Mod ID is the 2-4 digit number following the mod download name (like Custom Quickslots-3096-23-2-2-1647307597.zip 3096 is the mod ID)

**Comment by Pickysaurus** [comment]
Vortex already does this natively, you can also set the source manually to "Nexus Mods" and right-click on it. You'll see "Fix missing IDs" which uses the archive MD5 to get the mod data.

**Comment by effs** [comment]
> Vortex already does this natively, you can also set the source manually to "Nexus Mods" and right-click on it. You'll see "Fix missing IDs" which uses the archive MD5 to get the mod data. 

Oh, I'm not sure if I even noticed — as long as it's auto-added, that works. Rather not have any manual bits (although good to know that works too!)
